open-metadata-test/open-metadata-fvt/access-services-fvt/community-profile-fvt/build.gradle

```diff
@@ -0,0 +1,110 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright Contributors to the ODPi Egeria project.
+ */
+
+plugins {
+    // Include plug-in for starting a new process
+    id 'com.github.hesch.execfork'
```

Comment:
This is now outdated - recently in master we've changed to the psxpaul plugin. See https://github.com/odpi/egeria/commit/4834f05815ac439b5dd3b49ddb442dd8c9ccbc10 which also changes how the path is constructed

Reply:
best to review that commit for examples - as per the other fvts